cmd/run.go

```diff
@@ -211,6 +211,7 @@ func generateRunnerConfigFromInstance(instance *conf.InstanceConfig, perfConfig
 	runnerConfig.MessageOptions = instance.MessageOptions
 	runnerConfig.TokenOptions = instance.TokenOptions
 	runnerConfig.ContractOptions = instance.ContractOptions
+	runnerConfig.SubscriptionCoreOptions = instance.SubscriptionCoreOptions
```

Comment:
I think this one counts as an important configuration and should be documented in the readme